net_infrastructure/comware_5_2.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# (c) 2014, Patrick Galbraith <patg@patg.net>
+#
+# This file is part of Ansible
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: comware_5_2
+version_added: 0.1
+author: Patrick Galbraith
+short_description: Basic management of Comware-based 5.x Switches
+requirements: [ paramiko comware_5_2 (http://code.patg.net/comware_5_2.tar.gz)]
+description:
+    - Basic management of Comware 5.2-based Switches
+options:
+    developer-mode:
+        required: false
+        default: true
+        choices: [ true, false ]
+        description:
+            - Whether to set the switch into developer mode.
+    state:
+        required: false
+        default: present
+        choices: [ 'present', 'reboot' ]
```

Comment:
choices don't seem to make much sense, also prefer rebooted vs reboot as state. no absent?